### PR TITLE
PE-4533: Add Version Number to Login Screen

### PR DESCRIPTION
**Pull request by matibat**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/33p4auvocj9u8

### CODE REVIEW

File: lib/components/app_version_widget.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        if (info == null) {
+          logger.d('PackageInfo is null');
+          return const SizedBox(
+            height: 32,
+            width: 32,
+          );
+        }
+        final literalVersion =
+            kIsWeb ? info.version : '${info.version}+${info.buildNumber}';
+        logger.d('Version: $literalVersion');
```
Comment (thiagocarvalhodev) [inline lib/components/app_version_widget.dart:26]: Can we remove this log?

File: lib/components/app_version_widget.dart [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  const AppVersionWidget({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder(
+      future: PackageInfo.fromPlatform(),
+      builder: (BuildContext context, AsyncSnapshot<PackageInfo> snapshot) {
+        final info = snapshot.data;
+        if (info == null) {
+          logger.d('PackageInfo is null');
```
Comment (thiagocarvalhodev) [inline lib/components/app_version_widget.dart:18]: and this one?

**Review by thiagocarvalhodev — DISMISSED** [review]
LGTM.

**Review by thiagocarvalhodev — APPROVED** [review]
LGTM.

File: lib/components/app_version_widget.dart (hunk 1 above)
Comment (matibat) [reply lib/components/app_version_widget.dart]: Removed - d2f279cd87b85986a5f75da262579d4326401557

File: lib/components/app_version_widget.dart (hunk 2 above)
Comment (matibat) [reply lib/components/app_version_widget.dart]: Removed - d2f279cd87b85986a5f75da262579d4326401557

**Review by thiagocarvalhodev — APPROVED** [review]

**Pull request merged**